robot-command/src/main/java/org/obolibrary/robot/ExplainCommand.java

```diff
@@ -0,0 +1,146 @@
+package org.obolibrary.robot;
+
+import java.io.File;
+import java.io.Writer;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.util.Collections;
+import java.util.Set;
+import java.util.stream.Collectors;
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.Options;
+import org.semanticweb.owl.explanation.api.Explanation;
+import org.semanticweb.owlapi.apibinding.OWLManager;
+import org.semanticweb.owlapi.manchestersyntax.parser.ManchesterOWLSyntaxInlineAxiomParser;
+import org.semanticweb.owlapi.model.IRI;
+import org.semanticweb.owlapi.model.OWLAnnotationAssertionAxiom;
+import org.semanticweb.owlapi.model.OWLAxiom;
+import org.semanticweb.owlapi.model.OWLOntology;
+import org.semanticweb.owlapi.reasoner.OWLReasonerFactory;
+import org.semanticweb.owlapi.util.SimpleShortFormProvider;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Handles inputs and outputs for the {@link ExplainOperation}.
+ *
+ * @author <a href="mailto:balhoff@renci.org">Jim Balhoff</a>
+ */
+public class ExplainCommand implements Command {
+  /** Logger. */
+  private static final Logger logger = LoggerFactory.getLogger(ExplainCommand.class);
+
+  /** Store the command-line options for the command. */
+  private Options options;
+
+  public ExplainCommand() {
+    Options o = CommandLineHelper.getCommonOptions();
+    o.addOption("i", "input", true, "load ontology from a file");
+    o.addOption("I", "input-iri", true, "load ontology from an IRI");
+    o.addOption("r", "reasoner", true, "reasoner to use: ELK, HermiT, JFact");
+    o.addOption("a", "axiom", true, "the axiom to explain");
+    o.addOption("m", "max", true, "the maximum number of explanations to retrieve");
+    o.addOption("f", "format", true, "the explanation result format: MD, OFN, etc.");
```

Comment:
Could you add info in the docs on the valid output formats?

Reply:
This is kind of a mistake—I didn't meant to combine Markdown and OWL here (I think this line is from when I first started working on this). I will update to reflect the fact that the format is just the ordinary format choice all the other commands use.